src/routes/+page.svelte

```diff
@@ -15,6 +15,8 @@
 		})
 	);
 
+	const onSafari = navigator.userAgent.includes('Safari');
```

Comment:
@emikjackson when I try to run this locally, I see the following: "ReferenceError: navigator is not defined". Do you see this on your end too?

based on some quick googling, I wonder if this is because `navigator` is a client-side entity. I see that our landing page is rendered client-side, but not sure if that applies to the PromptWithSlider component as well? My initial guess would be that it's being rendered server side, which is why `nagivator` is undefined. 

Maybe we can change this to be CSR. Alternatively, I'm curious what you think of having the section be focused on clicking next/prev across all browsers so we don't need to distinguish between them. wdyt?

Reply:
update - I'm seeing the same error when I look at the netlify [logs](https://app.netlify.com/sites/dreamy-lokum-0293b5/deploys/673284c33472bf0008b3c3f3)!

Reply:
Yes! Sorry for not catching earlier, I'm surprised this wasn't showing for me when running and building locally. Just removed the safari check from the `section` so it won't cause issues during the prerender.

I left the Safari check in the `scrollToSection` function (which will only run on the browser) since I saw your comment afterwards, but I am open to also removing this so there's no difference across browsers. My only hesitation is that in general accessibility standards recommend not focusing on a non-interactive element, so I'm wondering if there's a benefit to jumping straight to the input that I'm not thinking about when doing my screenreader testing (if so, we may want to leave the focus-on-input for the other browsers).

I think @chelshaw originally added the `focus()` functionality - any preference on whether we just focus on the next `section` across browsers, or if we should avoid this behavior altogether?